ckanext/push_errors/plugin.py

```diff
@@ -85,4 +94,31 @@ def make_error_log_middleware(self, app, config):
     # IClick
 
     def get_commands(self):
-        return [push_errors_commands]
+        return [push_errors_commands, check_status_command]
+
+    def get_actions(self):
+        return {
+            'push_errors_enable': self.push_errors_enable,
+            'push_errors_disable': self.push_errors_disable,
+        }
+
+    def push_errors_enable(self, context, data_dict):
+        """Habilitar el envío de notificaciones"""
+        redis_client.set('push_errors:enabled', '1')
+        return {'status': 'enabled'}
+
+    def push_errors_disable(self, context, data_dict):
+        """Deshabilitar el envío de notificaciones"""
+        redis_client.set('push_errors:enabled', '0')
+        return {'status': 'disabled'}
+
+
+@click.command('push-errors-status', short_help='Verificar el estado de las notificaciones')
+def check_status_command():
+    status = redis_client.get('push_errors:enabled')
+    if status == b'1':
+        click.secho('Las notificaciones están HABILITADAS.', fg='green')
+    elif status == b'0':
+        click.secho('Las notificaciones están DESHABILITADAS.', fg='red')
+    else:
+        click.secho('El estado de las notificaciones es desconocido.', fg='yellow')
```

Comment:
`cli` comands must be defined in separated files, not here

Anyway, we don't need a command for this.
Delete it.
This could be a push-error settings and if it's available in `/ckan-admin/config`, better. This way we can enable or disable this.